### ISSUE TITLE
Video stream skipping/dropping frames

### ISSUE THREAD

**Issue by CUD** [aleganza/akuse#104]
Not sure if this is a common issue or just on my end, but the video seems to skip frames at times.

I'm sure the source of these streams so I can't compare to that but this has occurred for almost every show I have watched. Shows like Kaiju No. 8, Sentai Daishikakku and MHA.

Is this a known issue? Is there a way I can compare to the source stream?

**Comment by aleganza** [comment]
what happens more in details?

**Comment by CUD** [comment]
> what happens more in details?

This is an example below. How it drops frames when the camera pans down.
I see it in basically every episode I watch. It's especially noticeable when the camera pans like this.

![giphy](https://github.com/akuse-app/akuse/assets/8435061/2488f86b-59f5-4afa-89fa-4e55ced9bba0)

**Comment by aleganza** [comment]
is your pc a low end device? may I ask you configuration?

**Comment by CUD** [comment]
> is your pc a low end device? may I ask you configuration?

Nope, not low end. This is a desktop PC. 12700K CPU, 3080Ti, 32GB RAM.

I tend to watch videos on YouTube in Edge and haven't had this problem with vids there. This is only really an issue I've noticed in Akuse.

**Comment by aleganza** [comment]
this is really strange. even restarting your pc / restarting akuse solves anything?

**Comment by CUD** [comment]
> this is really strange. even restarting your pc / restarting akuse solves anything?

No difference after restarting my PC or the app as this has occurred over multiple days.

It might help if I can confirm where some of these streams are from and then I could compare to watching them in a web browser. While those newer shows I mentioned like MHA and others I can see this issue, I tried watching a couple of eps of Kill la Kill and this doesn't seem to be occurring. So somehow seems to be related to the source somehow.

**Comment by aleganza** [comment]
alright. you can find eng sources from gogoanime.

**Comment by aleganza** [comment]
does it happen with another version, for example the 1.1.0?

**Comment by CUD** [comment]
> alright. you can find eng sources from gogoanime.

I haven't been able to replicate this issue when streaming from that site in Edge.

> does it happen with another version, for example the 1.1.0?

I tried that version again to confirm, the issue still occurs for me on that version also.

**Comment by aleganza** [comment]
I recently noticed that the video player actually uses relatively a lot of gpu, maybe this could be related but you have a gpu even more powerful than mine so I really couldn'think about anything.
when you play an episode, what specs are you shown in the task manager? (or in the linux corresponding program if you're not on windows)
I'd like know about ram, cpu and gpu values

**Comment by CUD** [comment]
> I recently noticed that the video player actually uses relatively a lot of gpu, maybe this could be related but you have a gpu even more powerful than mine so I really couldn'think about anything. when you play an episode, what specs are you shown in the task manager? (or in the linux corresponding program if you're not on windows) I'd like know about ram, cpu and gpu values

I think I've found what is causing this.. but for reference I'm on W11. CPU tends to stay under 1%, GPU will stay around 10%, RAM will be around 300MB if I only load one video and then another 100MB+ for each video after that.

However, this seems to be resolved now in my tests. It seems a global setting in my NVIDIA control panel was causing this, the setting 'background application max frame rate'. I have this set to 20 fps for background apps. After turning this off for akuse I'm not longer seeing these frame drops.. it looks like at some point I had already turned this off for Edge which would explain why it wasn't occurring when watching videos in browser.

It didn't occur to me that it would be this at first as it wouldn't normally apply if the application is the active window, it's more if I were to alt-tab out of a game or app so that it's not in focus that it should restrict the framerate. In the case of akuse it seems to apply regardless.

Thanks for looking into it though. I'll mark this as closed.
